pkg/cmd/admin/router/router.go

```diff
@@ -438,7 +440,8 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 			return fmt.Errorf("router could not be created; you must specify a service account with --service-account")
```

Comment:
You don't have to run a router exposed to the outside - that's actually
something this pr should fix.  Default should be hostNetwork, but host port
and pod network only are valid

On Oct 21, 2015, at 2:50 AM, Bo Meng <notifications@github.com> wrote:

In pkg/cmd/admin/router/router.go
<https://github.com/openshift/origin/pull/5265#discussion_r42588836>:

> @@ -438,7 +440,8 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
>  			return fmt.Errorf("router could not be created; you must specify a service account with --service-account")

It will also need to use the host ports when using --host-network=false.
Which will also need the --service-account option.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5265/files#r42588836>.